Utils.py

```diff
@@ -422,6 +422,10 @@ def get_text_between(text: str, start: str, end: str) -> str:
     return text[text.index(start) + len(start): text.rindex(end)]
 
 
+def get_text_after(text: str, start: str) -> str:
+    return text[text.index(start) + len(start):]
+
+
```

Comment:
this seems to be unused in this PR?

Reply:
Yeah... the associated Factorio PR uses it. Apparently one change went into the wrong commit. This is still not too bad to have in core.